admin/spec/controllers/spree/admin/refund_resons_controller_spec.rb

```
@@ -0,0 +1,85 @@
+require 'spec_helper'
+
+RSpec.describe Spree::Admin::RefundReasonsController, type: :controller do
```

RuboCop (1.75.5)

[convention] Spec path should end with `spree/admin/refund_reasons_controller*_spec.rb`.

(RSpec/SpecFilePathFormat)